tests/garage/envs/test_resize_env.py

```diff
@@ -0,0 +1,11 @@
+import unittest
+
+import gym
+
+from garage.envs.wrappers import Resize
+
+
+class TestResize(unittest.TestCase):
+    def test_resize_output_shape(self):
+        env_r = Resize(gym.make("Breakout-v0"), width=84, height=84)
```

Comment:
Use mocks to avoid interacting with gym

https://semaphoreci.com/community/tutorials/getting-started-with-mocking-in-python